shuttle/src/runtime/task/mod.rs

```diff
@@ -233,6 +237,7 @@ impl Task {
             span_stack,
             local_storage: StorageMap::new(),
             tag: None,
+            backtrace: Backtrace::disabled(),
```

Comment:
The documentation on `Backtrace::disabled` is super confusing, because it states "Forcibly captures a disabled backtrace, regardless of environment variable configuration."

But that seems like a cut-n-paste error, because that's not what the implementation does: https://doc.rust-lang.org/beta/src/std/backtrace.rs.html#319-321
🤷‍♂️

Reply:
Yeah I also notice that